x-pack/plugins/security_solution/public/detections/components/alerts_table/alerts_utility_bar/index.tsx

```diff
@@ -59,18 +61,24 @@ const BuildingBlockContainer = styled(EuiFlexItem)`
   padding: ${({ theme }) => `${theme.eui.paddingSizes.xs}`};
 `;
 
+const ThreatMatchContainer = styled(EuiFlexItem)`
```

Comment:
I don't know if it's coming from this component, but I found the amount of vertical space between the two existing filters surprising/excessive: 

<kbd><img width="322" alt="Detections_-_Kibana" src="https://user-images.githubusercontent.com/657252/113806389-d0f26f80-9727-11eb-8557-a38b57fe88c9.png"></kbd>

Reply:
reduced gutter size from default "l" to "m", looks a bit better
<img width="398" alt="Screen Shot 2021-04-07 at 12 33 24 PM" src="https://user-images.githubusercontent.com/18617331/113902091-88759900-979d-11eb-8d26-ea35f771dfa3.png">